src/App.tsx

```diff
@@ -1,7 +1,48 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  hasPressedKey: boolean,
+  pressedKey: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    hasPressedKey: false,
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleDocumentPressKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleDocumentPressKey);
+  }
+
+  handleDocumentPressKey = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key, hasPressedKey: true });
+  };
+
+  render() {
+    const {
+      pressedKey,
+      hasPressedKey,
+    } = this.state;
+
+    return (
+      <div className="App">
+
+        {hasPressedKey ? (
+          <p className="App__message">
+            {`The last pressed key is [${pressedKey}]`}
+          </p>
+        ) : (
+          <p className="App__message">
+            Nothing was pressed yet
+          </p>
+        )}
```

Comment:
You have a duplicate wrapper to the text, transfer the condition to the text only